dashboard/test/integration/registration/set_email_test.rb

```diff
@@ -32,9 +32,9 @@ class SetEmailTest < ActionDispatch::IntegrationTest
       end
 
       assert_response :unprocessable_entity
-      assert_equal ['Authentication options is invalid', 'Email is invalid'], assigns(:user).errors.full_messages
+      assert_equal ['Authentication options email is invalid', 'Email is invalid'], assigns(:user).errors.full_messages
```

Comment:
This and line 37 are changed as the error message format has changed, due to `accepts_nested_attributes_for :authentication_options` on the user model being added.